src/ActionBuilder/ProductType/ChangeOrderOfAttributes.php

```diff
@@ -42,21 +42,33 @@ public function createUpdateActions(
         array $oldData,
         $sourceObject
     ): array {
-        $actions = [];
+        if (!class_exists(ProductTypeChangeAttributeOrderByNameAction::class, true)) {
+            return [];
```

Comment:
I would really prefer just one return in this method.

Reply:
Why? In my opinion, an early return here leads to a better readable/understandable method.
Removing the early return would result in an additional nesting which makes the method unclear.

Putting "Guarding Clauses" in top of a method is a good way to signal what is necessary for the method to actually run instead of hiding it in the body.